.github/workflows/differential-shellcheck.yml

```diff
@@ -26,6 +26,6 @@ jobs:
           fetch-depth: 0
 
       - name: Differential ShellCheck
-        uses: redhat-plumbers-in-action/differential-shellcheck@60360c0fb283149ed03ad16b66257a967c3e5231
+        uses: redhat-plumbers-in-action/differential-shellcheck@v3
```

Comment:
Please use `a14889568f6210b361eb29e16f3b07f512fca846` to comply with repository convention, where actions are pinned to SHA1.

https://github.com/redhat-plumbers-in-action/differential-shellcheck/commit/a14889568f6210b361eb29e16f3b07f512fca846